charts/images.yaml

```diff
@@ -253,3 +253,7 @@ images:
   sourceRepository: github.com/gardener/apiserver-proxy
   repository: eu.gcr.io/gardener-project/gardener/apiserver-proxy
   tag: "v0.1.0"
```

Comment:
Can we update this to `v0.2.0` as well?
```suggestion
  tag: "v0.2.0"
```

Reply:
There are no functional changes between `0.1.0` and `0.2.0`. This will cause unneeded rollout of the daemonset.

Reply:
OK